frontend/src/router/guards.js

```diff
@@ -36,10 +36,14 @@ function setLoading (store, value) {
 
 function ensureConfigurationLoaded (store) {
   return async function (to, from, next) {
+    const meta = to.meta || {}
```

Comment:
If we move `ensureConfigurationLoaded` inside of `ensureDateLoaded` https://github.com/gardener/dashboard/blob/42fca41d290705e4bbe608376166f3bb743c34ae/frontend/src/router/guards.js#L101 we don't need this. And I think now we can consider config as data.

Reply:
✅